client/src/components/profileInfoEdit/ProfileDeleteModal.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+import React from 'react';
+ import toast from 'react-hot-toast';
+ import { Button, Modal } from 'react-bootstrap'; 
+ import { useSelector, useDispatch } from 'react-redux';
+ import { logout } from '../../actions/auth';
+ import { deleteTeam ,deleteGHPages} from '../../actions/team';
+
+
+  const ProfileDeleteModal = ({ deleteAlert, setdeleteAlert }) => {
+   
+   const {teamId} = useSelector((state) => state.team,);
+   const dispatch = useDispatch();
+
+   const fullDelete = () => {
+    
+    const access_token = localStorage.getItem('GH_access_token');
+    if(access_token===null) {
+      toast.error('Log in with github account')
+    } 
+    else{
+      try{
+      dispatch(deleteGHPages(teamId,access_token))
+      dispatch(deleteTeam(teamId))
+      dispatch(logout())
+      } catch(error){
+        toast.error('GitHub Pages doesnt exist')
+      }
```

Comment:
Interesting that the SA didn't catch these indents and the missing semicolons. Thoughts @vgoh1 ?

Reply:
We'll need to fix the SA if that's the case. The style is very inconsistent here.

Reply:
If it's conflicting with `prettier` rules, which may be more flexible, we may need to change prettier instead.

Reply:
Yeah, let me look into why the linter isn't catching the formatting + the prettier issue. I personally want to make prettier format according to airbnb because it's faster doing the shortcut to format than running npm run lint.

Reply:
@cyr17 Can you run `npm run lint` in the /api and /client and fix the errors that come up? They come up there for me. The formatting stuff can be fixed using `npm run lint -- --fix`.

Reply:
This is wierd xp,
npm run lint on /api
![image](https://user-images.githubusercontent.com/73247453/131655264-0a7cde17-8e7d-4d26-a001-5dca4fd06d83.png)

npm run lint on /client
![image](https://user-images.githubusercontent.com/73247453/131655106-705813f8-cc55-40f6-b2c7-a038983565bc.png)